hanoi_tower/Main.c

```diff
@@ -0,0 +1,198 @@
+#include <stdio.h>
```

Comment:
@leonardo-anjos Kindly add a description and a sample case for your code.

Reply:
ok, thanks